gremlin-test/features/map/Select.feature

```diff
@@ -514,4 +514,20 @@ Feature: Step - select()
     Then the result should be unordered
       | result |
       | d[2].l |
-      | d[2].l |
\ No newline at end of file
+      | d[2].l |
+
+  Scenario: g_V_asXaX_groupXmX_by_byXbothE_countX_barrier_selectXmX_selectXselectXaXX
+    Given the modern graph
+    And the traversal of
+      """
+      g.V().as("a").group("m").by().by(__.bothE().count()).barrier().select("m").select(__.select("a"))
+      """
+    When iterated to list
+    Then the result should be unordered
+      | result |
+      | d[3].l |
+      | d[1].l |
+      | d[3].l |
+      | d[3].l |
+      | d[1].l |
+      | d[1].l |
```

Comment:
Could you please add one more test to include the other method you added - `select(Pop,Traversal)` ? I think it's worth exercising all these top level steps on the odd chance that something stops working somewhere.